app/components/signum/notification/component.html.slim

```diff
@@ -1,24 +1,35 @@
-.signum-notification.hidden data-notification-timeout=@data[:timeout] data-controller="notification"
-  .p-4
-    .flex.items-start
-      .flex-shrink-0
-        - if @signal.kind == "success"
-          svg.signum-notification-icon-success[xmlns="http://www.w3.org/2000/svg" fill="none" viewbox="0 0 24 24" stroke-width="1.5" stroke="currentColor" aria-hidden="true"]
-            path[stroke-linecap="round" stroke-linejoin="round" d="M9 12.75L11.25 15 15 9.75M21 12a9 9 0 11-18 0 9 9 0 0118 0z"]
-        - elsif @signal.kind == "error"
-          svg.signum-notification-icon-error[xmlns="http://www.w3.org/2000/svg" fill="none" viewbox="0 0 24 24" stroke-width="1.5" stroke="currentColor"]
-            path[stroke-linecap="round" stroke-linejoin="round" d="M9.75 9.75l4.5 4.5m0-4.5l-4.5 4.5M21 12a9 9 0 11-18 0 9 9 0 0118 0z"]
-        - else
-          svg.svg.signum-notification-icon-info[xmlns="http://www.w3.org/2000/svg" fill="none" viewbox="0 0 24 24" stroke-width="1.5" stroke="currentColor" aria-hidden="true"]
-            path[stroke-linecap="round" stroke-linejoin="round" d="M11.25 11.25l.041-.02a.75.75 0 011.063.852l-.708 2.836a.75.75 0 001.063.853l.041-.021M21 12a9 9 0 11-18 0 9 9 0 0118 0zm-9-3.75h.008v.008H12V8.25z"]
-      .ml-3.w-0.flex-1.pt-0.5
-        p.signum-notification-title
-          = sanitize @signal.title
-        p.signum-notification-body
-          = sanitize @signal.text
-      .ml-4.flex.flex-shrink-0
-        button.signum-notification-button-close[type="button" data-action="notification#close"]
-          span.sr-only
-            | Close
-          svg.h-5.w-5[xmlns="http://www.w3.org/2000/svg" viewbox="0 0 20 20" fill="currentColor" aria-hidden="true"]
-            path[d="M6.28 5.22a.75.75 0 00-1.06 1.06L8.94 10l-3.72 3.72a.75.75 0 101.06 1.06L10 11.06l3.72 3.72a.75.75 0 101.06-1.06L11.06 10l3.72-3.72a.75.75 0 00-1.06-1.06L10 8.94 6.28 5.22z"]
+.signum-notification class=('hidden' if @data[:age] == 'new') data-notification-timeout-value=@data[:timeout] data-notification-sticky-value=@signal.sticky data-controller="notification" id="notifications_#{@signal.signalable_id}_#{@signal.id}"
+  .p-4.flex.w-0.flex-1.items-center
+    .flex-shrink-0
+      - if @signal.icon.present?
```

Comment:
See below, it would be more developer friendly to use `icon: 'fad fa-apple-whole'`